cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -157,12 +169,14 @@ protected CloudControllerRestClientImpl() {
     }
 
     public CloudControllerRestClientImpl(URL controllerUrl, CloudCredentials credentials, RestTemplate restTemplate,
-        OAuthClient oAuthClient, LoggregatorClient loggregatorClient) {
-        this(controllerUrl, credentials, restTemplate, oAuthClient, loggregatorClient, null);
+        OAuthClient oAuthClient, LoggregatorClient loggregatorClient, CloudFoundryOperations v3OperationsClient,
+        CloudFoundryClient v3Client) {
```

Comment:
Wouldn't be better if the variables v3* are named without the v3, if we are with the assumption that everything is v3